### DISCUSSION TITLE
Poll: Help Us Choose Our New Name! 🚀

### DISCUSSION THREAD

**Discussion by tjbck** [open-webui/open-webui#601] [Polls]
We're rebranding to avoid confusion with official Ollama products. Choose our new name:

1. **One-webui**: Unity & integration of multiple LLMs.
2. **Open-webui**: Openness & flexibility for various LLMs.

Your input shapes our future!

**Comment by justinh-rahb** [comment]
@tjbck I'm sorry but neither of these really feel great. If we need to change, we should create something really special and unique. I don't have any great ideas to suggest, I just know we could do better if we took our time to come up with some candidates I think.

**Comment by tjbck** [comment]
@justinh-rahb I agree, let's take our time to come up with some candidates first. it seems like the result is 50/50 anyway which really doesn't help 😅. I'll close this poll for now.
